irods/message/__init__.py

```diff
@@ -82,8 +92,10 @@ class BadXMLSpec(RuntimeError): pass
 
 _XML_strings = { k:v for k,v in vars(XML_Parser_Type).items() if k.endswith('_XML')}
 
+_default_XML = os.environ.get('PYTHON_IRODSCLIENT_DEFAULT_XML',globals().get('_default_XML'))
+
+#print('dXML',_default_XML,'from env')#dwm
```

Comment:
let's remove dwm - okay to have a debug comment, but let's document as such.

Reply:
Comment is removed.